GEMINI.md

```
@@ -0,0 +1,266 @@
+### 4.1 Unit & E2E
+
+- **Unit tests:** `make test`. ([vald][1])
+
+- **E2E (classic flow):**
+  1. Prepare Fashion-MNIST dataset
+  2. Start local k3d
+  3. Deploy Vald via Make target
+  4. Run E2E tests
+  5. Delete the cluster
+
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)